### DISCUSSION TITLE
Better Support for Symfonium App

### DISCUSSION THREAD

**Discussion by HolyMacarony** [navidrome/navidrome#2398] [Ideas]
Hello! 
I'd like to ask whether it is planned to support the Tag/filter Features of Symfonium, which IMHO is the best client for android by far.
Here are the infos: https://support.symfonium.app/t/wiki-supported-features-by-media-providers/1178

thank you very much!

**Comment by deluan** [comment]
Symfonium communicates with Navidrome using the Subsonic API. We are collaborating to improve it with the OpenSubsonic project, and this may be supported in the future. Currently there's nothing I can do in Navidrome to support this.
